syncer/handle_error_test.go

```diff
@@ -62,7 +63,7 @@ func (s *testSyncerSuite) TestHandleError(c *C) {
 			},
 		}
 	)
-
+	mockDB := conn.InitMockDB(c)
```

Comment:
this may cause unstable test, bacause syncer may flush checkpoint when exit

Reply:
actually, it doesn't. If an error is reported here, there will only be a warnning